torchtune/datasets/_stack_exchanged_paired.py

```diff
@@ -1,46 +0,0 @@
-# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
this whole time we've used "stack exchange**d**"? lol

Reply:
I wouldn't have noticed this if not for the filename diff, no shade meant

Reply:
case in point: I have to look up how to `git reset --hard` at least three times a week on stack overflowed

Reply:
the name was bothering me for a long time 😅

Reply:
"overflowed" lol